rdkit/ML/Cluster/Murtagh.py

```diff
@@ -38,73 +41,73 @@
 
 
 def _LookupDist(dists, i, j, n):
-  """ *Internal Use Only*
+    """ *Internal Use Only*
 
-   returns the distance between points i and j in the symmetric
-   distance matrix _dists_
+     returns the distance between points i and j in the symmetric
+     distance matrix _dists_
 
-  """
-  if i == j:
-    return 0.0
-  if i > j:
-    i, j = j, i
-  return dists[j * (j - 1) / 2 + i]
+    """
+    if i == j:
+        return 0.0
+    if i > j:
+        i, j = j, i
+    return dists[j * (j - 1) / 2 + i]
```

Comment:
I expect this was written for py2, we should use // for integer division I’m assuming.

Reply:
yeah, this is ancient code. Nice catch.
It's fixed.